packages/storybook/src/nimble/anchor-tabs/anchor-tabs-matrix.stories.ts

```diff
@@ -36,10 +62,15 @@ const component = (
     widthValue: WidthState
 ): ViewTemplate => html`
     <${anchorTabsTag} activeid="tab1" style="padding: 15px;${widthValue ? ` width: ${widthValue};` : ''}">
-        ${when(() => toolbar, html`
+        ${when(() => toolbar.showToolbar, html`
```

Comment:
Can we add text labels that describe the cases? There are enough permutations that it's hard to follow without some labels